plugins/base/src/test/kotlin/content/HighlightingTest.kt

```diff
@@ -71,9 +71,10 @@ class HighlightingTest : BaseAbstractTest() {
                     assertTrue(children?.get(it.first)?.style?.contains(it.second) == true)
                 val annotation = children?.first()?.children?.first()
 
-                assertTrue(annotation?.children?.get(0)?.style?.contains(TokenStyle.Annotation) == true)
-                assertTrue(annotation?.children?.get(1)?.children?.first()?.style?.contains(TokenStyle.Annotation) == true)
+                val annotationStyles = setOf(TokenStyle.Annotation, TokenStyle.Builtin)
```

Comment:
What does `TokenStyle.Builtin` mean?
I am not sure the output-specific problem should be fixed here, in the content model rather than in the render stage.

Reply:
> What does TokenStyle.Builtin mean?

It's a good question, you were the one that added the type :) It was already in Public API, so I just used it.

> I am not sure the output-specific problem should be fixed here, in the content model rather than in the render stage.

Sure, I can fix it for html-only, but let's then delete `TokenStyle.Builtin` entry so there's no confusion?

Reply:
> It's a good question, you were the one that added the type :) It was already in Public API, so I just used it.

I hoped somebody tells me what it means)
You can remove it since it is not used anywhere.